problems/arrayMethods.js

```diff
@@ -5,7 +5,16 @@
  * @returns {number} Sum of all numbers
  */
 
-const sumArray = () => {};
+const sumArray = (array) => {
+  //take in an array
+  let sum = 0;
+
```

Comment:
To make your functions cleaner and more readable, remove these empty lines from the code.